packages/shared/src/env.ts

```diff
@@ -39,6 +39,10 @@ const EnvSchema = z.object({
     .positive()
     .default(60 * 10),
   SALT: z.string().optional(), // used by components imported by web package
+  LANGFUSE_LOG_LEVEL: z
+    .enum(["trace", "debug", "info", "warn", "error", "fatal"])
+    .optional(),
+  LANGFUSE_LOG_FORMAT: z.enum(["text", "json"]).default("text"),
```

Comment:
How do these environment variables work? `LANGFUSE_LOG_LEVEL` is also/still defined on the worker. Do shared environments bubble up, i.e. shall I remove the reference in the worker env.ts?

Reply:
Yes, this bubbles up. Yes, removing this from worker makes sense. can you also add this to `.env.prod.example`?

Reply:
Clarified offline with Marc. Will remove it from the worker and leave it in the shared package only. Corresponding doc adjustment is prepared in https://github.com/langfuse/langfuse-docs/pull/789.